suite-common/message-system/schema/config.schema.v1.json

```diff
@@ -367,7 +368,7 @@
             "title": "Localization",
             "description": "A multilingual text localization.",
             "type": "object",
-            "required": ["en-GB", "en", "es", "cs", "ru", "ja"],
```

Comment:
Was it intentional to remove `"ru", "ja"` from the list, making them non-required?

Just making sure, because in PR description you wrote you are only adding :slightly_smiling_face:

Reply:
Yes, these are community languages, so they are not required. Only official lngs should be imho.